src/integrationTest/java/org/opensearch/test/framework/cluster/LocalCluster.java

```diff
@@ -531,4 +533,7 @@ public TestCertificates getTestCertificates() {
         return testCertificates;
     }
 
+    public void setPrimaryNode(String primaryNode) {
```

Comment:
why need primary Node?

Reply:
During creation of high level rest client available nodes were probed, cluster managers would have highest priority, but data nodes were also checked. On top of that nodes were rotated which caused inconsistency amount of transport audit logs. This method is used to force one node to be used as rest client